Marlin/language.h

```diff
@@ -242,6 +242,7 @@
 #define MSG_DEBUG_INFO                      "DEBUG INFO ENABLED"
 #define MSG_DEBUG_ERRORS                    "DEBUG ERRORS ENABLED"
 #define MSG_DEBUG_DRYRUN                    "DEBUG DRYRUN ENABLED"
+#define MSG_DEBUG_COMMUNICATION             "DEBUG COMMUNICATION ENABLED"
 #define MSG_DEBUG_LEVELING                  "DEBUG LEVELING ENABLED"
```

Comment:
As Marlin is starting to get too huge for 128K boards, we should try to consolidate strings like this which are never translated. Hosts don't care about these, so I recommend rewording to:
```cpp
  #define MSG_DEBUG_OFF                       "DEBUG OFF"
  #define MSG_DEBUG_PREFIX                    "DEBUG ENABLED: "
  #define MSG_DEBUG_INFO                      "INFO"
  #define MSG_DEBUG_ERRORS                    "ERRORS"
  #define MSG_DEBUG_DRYRUN                    "DRYRUN"
  #define MSG_DEBUG_COMMUNICATION             "COMMUNICATION"
  #define MSG_DEBUG_LEVELING                  "LEVELING"
```
Then in the `M111` function, perhaps a single line of output:
```cpp
static const char *debug_strings[] = {
  PSTR(MSG_DEBUG_ECHO),
  PSTR(MSG_DEBUG_INFO),
  PSTR(MSG_DEBUG_ERRORS),
  PSTR(MSG_DEBUG_DRYRUN),
  PSTR(MSG_DEBUG_COMMUNICATION),
  #if ENABLED(DEBUG_LEVELING_FEATURE)
    PSTR(MSG_DEBUG_LEVELING)
  #endif
};

SERIAL_ECHO_START;
if (marlin_debug_flags) {
  SERIAL_ECHOPGM(MSG_DEBUG_PREFIX);

  uint8_t comma = 0;
  for (uint8_t i = 0; i < COUNT(debug_strings); i++) {
    if (TEST(marlin_debug_flags, i)) {
      if (comma++) SERIAL_ECHOPGM(PSTR(", "));
      SERIAL_ECHOPGM(debug_strings[i]);
    }
  }
  SERIAL_EOL;
}
else {
  SERIAL_ECHOLNPGM(MSG_DEBUG_OFF);
}
```